### PR TITLE
Enable debugging plugin goal

### PR DESCRIPTION
**Pull request by Eskibear**
1. create a terminal with debugging jvm args injected by MAVEN_OPTS
2. execute mvn goal
3. attach debugger

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -25,4 +26,5 @@
     "configuration.maven.terminal.favorites": "Specify pre-defined favorite commands to execute.",
     "configuration.maven.terminal.favorites.alias": "A short name for the command.",
     "configuration.maven.terminal.favorites.command": "Content of the favorite command."
+
```
Comment (jdneo) [inline package.nls.json]: Redundant empty line

File: src/handlers/debugHandler.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+async function debug(pomPath: string, command: string): Promise<void> {
+    const freePort: number = await getPort();
+    const mavenOpts: string = [
+        Settings.getEnvironment().MAVEN_OPTS, // user-setting MAVEN_OPTS
+        `-Xdebug -Xrunjdwp:transport=dt_socket,server=y,suspend=y,address=${freePort}` // MAVEN_DEBUG_OPTS
+    ].filter(Boolean).join(" ");
+    const sessionId: string = createUuid().substr(0, 6);
+    const debugTerimnal: vscode.Terminal = await executeInTerminal(command, pomPath, {
+        name: `mvnDebug ${sessionId}: ${command}`,
+        env: { MAVEN_OPTS: mavenOpts }
```
Comment (jdneo) [inline src/handlers/debugHandler.ts]: Push the `process.env` into the `env` here as well?

File: src/handlers/debugHandler.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    });
+    const debugConfig: vscode.DebugConfiguration = {
+        type: "java",
+        name: `Maven (Attach) - ${sessionId}`,
+        request: "attach",
+        hostName: "localhost",
+        port: freePort,
+        terminalName: debugTerimnal.name
```
Comment (jdneo) [inline src/handlers/debugHandler.ts]: Need to provide project name

Comment (Eskibear) [reply src/handlers/debugHandler.ts]: Just discussed with debugger extension maintainer, I would rather leave it empty, debugger will try resolving it.

File: src/handlers/debugHandler.ts (hunk 2 above)
Comment (Eskibear) [reply src/handlers/debugHandler.ts]: on creating a terminal, vscode already ensures that. 
```ts
		/**
		 * Object with environment variables that will be added to the VS Code process.
		 */
		env?: { [key: string]: string | null };
```

**Pull request merged**